cartridge-cli.lua

```diff
@@ -2726,6 +2749,13 @@ local function check_if_deprecated_build_flow_is_ised(app_path)
         end
     end
 
+    if deprecated_build_flow_is_ised then
```

Comment:
I believe that moving of this part is redundant

Reply:
This warning message was printed only on application packing. I sure that we should warn on building application too.

Reply:
Ok, yes